### PR TITLE
Deploy letsencrypt certificates (clean branch)

### PR DESCRIPTION
**Pull request by mvdriel**

### CODE REVIEW

**Comment by mvdriel** [comment]
@tersmitten Would it be possible to merge this pull request when I make it up-to-date?

**Comment by tersmitten** [comment]
Yes

**Comment by mvdriel** [comment]
@tersmitten Nice! I think it is up-to-date now.

File: tasks/letsencrypt.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    group: root
+    mode: 0755
+  tags:
+    - haproxy-letsencrypt-ocsp-deploy
+
+- name: letsencrypt | configure (cron) job for OCSP deploy
+  cron:
+    name: haproxy-letsencrypt-ocsp-deploy
+    job: "{{ haproxy_letsencrypt_ocsp_deploy }}"
+    state: "{{ haproxy_letsencrypt_ocsp_deploy_job.state | default('absent') }}"
```
Comment (tersmitten) [inline tasks/letsencrypt.yml:27]: Most of the times, no `state` means `present`. Is this on purpose?

Comment (mvdriel) [reply tasks/letsencrypt.yml:27]: `present` is better, but that will create the job when there is no (custom) configuration. Do you want that?

Comment (tersmitten) [reply tasks/letsencrypt.yml:27]: Probably, you're right

**Pull request merged**